GNUmakefile

```diff
@@ -60,38 +62,40 @@ lib/renderers/wayland/wlr-layer-shell-unstable-v1.h:
 lib/renderers/wayland/wlr-layer-shell-unstable-v1.c:
 	wayland-scanner code < $(subst .c,.xml,$@) > $@
 
+xdg-shell.a: private override CPPFLAGS += `pkg-config --cflags-only-I wayland-client`
 xdg-shell.a: lib/renderers/wayland/xdg-shell.c
+wlr-layer-shell.a: private override CPPFLAGS += `pkg-config --cflags-only-I wayland-client`
 wlr-layer-shell.a: lib/renderers/wayland/wlr-layer-shell-unstable-v1.c lib/renderers/wayland/wlr-layer-shell-unstable-v1.h
-bemenu-renderer-wayland.so: private override LDLIBS += `pkg-config --libs wayland-client cairo pango pangocairo xkbcommon`
-bemenu-renderer-wayland.so: private override CPPFLAGS += `pkg-config --cflags-only-I wayland-client cairo pango pangocairo xkbcommon`
+bemenu-renderer-wayland.so: private override LDLIBS += `pkg-config --libs wayland-client cairo pango pangocairo xkbcommon` `pkg-config --silence-errors --libs epoll-shim`
+bemenu-renderer-wayland.so: private override CPPFLAGS += `pkg-config --cflags-only-I wayland-client cairo pango pangocairo xkbcommon` `pkg-config --silence-errors --cflags-only-I epoll-shim`
```

Comment:
This is fair enough, but I'd prefer approach of documenting and providing these extras through command line.
`make LDLIBS=$(pkg-config --libs epoll-shim) CPPFLAGS=$(pkg-config --cflags-only-I epoll-shim)`